File: prog-multi-dispmoviles/TareaDisMoviles/app/src/main/java/com/example/tareadismoviles/MainActivity.java
package com.example.tareadismoviles;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;

import com.example.tareadismoviles.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final EditText editText1 = findViewById(R.id.editText1);
        final EditText editText2 = findViewById(R.id.editText2);
        final EditText editText3 = findViewById(R.id.editText3);
        Button concatenateButton = findViewById(R.id.concatenateButton);

        concatenateButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String text1 = editText1.getText().toString();
                String text2 = editText2.getText().toString();
                String text3 = editText3.getText().toString();
                String concatenatedText = text1 + " " + text2 + " " + text3;
                Toast.makeText(MainActivity.this, concatenatedText, Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: prog-multi-dispmoviles/FinalExtra/app/src/main/java/com/example/finalextra/MainActivity.kt
package com.example.finalextra

import android.annotation.SuppressLint
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    @SuppressLint("MissingInflatedId")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val usuario = findViewById<EditText>(R.id.campo_Nick)
        val contraseña = findViewById<EditText>(R.id.campo_contraseña)
        val telefono = findViewById<EditText>(R.id.campo_telefono)
        val enviar = findViewById<Button>(R.id.Boton_enviar)

        enviar.setOnClickListener{
            if (usuario.getText().toString().isEmpty()||contraseña.getText().toString().isEmpty()){
                Toast.makeText(this, "campo obligatorio vacio", Toast.LENGTH_LONG).show();
            }else{
               if (usuario.getText().toString() != "luis"){
                   Toast.makeText(this, "el campo Nick no es correcto", Toast.LENGTH_LONG).show();
               }else{
                   if (contraseña.getText().toString() != "admin"){
                       Toast.makeText(this, "el campo Contraseña no es correcto", Toast.LENGTH_LONG).show();
                   }else{
                       startActivity(Intent(this, Pantalla2::class.java))
                   }
               }
            }
        }
    }
}